### PR TITLE
Make header position configurable

### PR DESCRIPTION
**Pull request by AbdullahM0hamed**
Show some love to us folk who prefer the headers to be at the top.

### CODE REVIEW

**Comment by AbdullahM0hamed** [comment]
Perhaps you will find this screenshot useful, the top is when the value of 'header_position' is set to 'top', and the bottom is when it is set to bottom (which is the default).

File: anime_downloader/config.py [hunk 1]
```diff
@@ -21,6 +21,9 @@
         'external_downloader': '',
         'aria2c_for_torrents': False,
     },
+    'search': {
```
Comment (vn-ki) [inline anime_downloader/config.py:24]: don't need a new section. put it in `dl` maybe?

File: anime_downloader/util.py [hunk 2]
```diff
@@ -68,7 +75,11 @@ def format_search_results(search_results):
     table = [(i+1, v.title, v.pretty_metadata)
              for i, v in enumerate(search_results)]
     table = tabulate(table, headers, tablefmt='psql')
-    table = '\n'.join(table.split('\n')[::-1])
+    try:
+        layout = positions[Config._CONFIG["search"]["header_position"]]
+    except KeyError:
+        raise KeyError(f"The value '{Config._CONFIG['search']['header_position']}' is not a valid position for the headers")
```
Comment (vn-ki) [inline anime_downloader/util.py:81]: Warn, don't error

**Comment by AbdullahM0hamed** [comment]
Done

File: anime_downloader/util.py [hunk 3]
```diff
@@ -68,7 +75,12 @@ def format_search_results(search_results):
     table = [(i+1, v.title, v.pretty_metadata)
              for i, v in enumerate(search_results)]
     table = tabulate(table, headers, tablefmt='psql')
-    table = '\n'.join(table.split('\n')[::-1])
```
Comment (vn-ki) [inline anime_downloader/util.py:71]: Needs a default `layout`

Comment (AbdullahM0hamed) [reply anime_downloader/util.py:71]: Gave it a default value of -1 on line 79 now.

**Comment by vn-ki** [comment]
Conflicts

**Comment by AbdullahM0hamed** [comment]
I believe that she be solved now

File: anime_downloader/config.py [hunk 4]
```diff
@@ -23,6 +23,7 @@
         'selescrape_browser': None,
         'selescrape_browser_executable_path' : None,
         'selescrape_driver_binary_path' : None,
+        'header_position': 'bottom'
```
Comment (vn-ki) [inline anime_downloader/config.py:26]: I guess this should be in a new section `common`, but we can do that later.

**Comment by vn-ki** [comment]
Reverted. You have broken the build.

**Pull request merged**